sources/packages/api/src-sql/ArchiveDB/SetupDB.sql

```diff
@@ -0,0 +1,10 @@
+-- Individual table
+CREATE TABLE IF NOT EXISTS individual ( 
+  individual_idx serial primary key,
+  sin numeric(22,9),
+  permanent_disability_flg varchar(1),
+  date_of_birth timestamp without time zone
+);
+
+-- Insert Test Item
+INSERT INTO individual(sin,permanent_disability_flg,date_of_birth) VALUES (123456799.000000000,'Y','2000-03-11 00:00:00');
```

Comment:
I would recommend that the test item should be inserted during the E2E test and removed once the E2E is done.

Reply:
This whole items are available for local env only. We can keep them

Reply:
For remote e2e testing I've no idea how to proceed.

Reply:
In my option, if it belongs to the E2E test and if is just gonna be used for the E2E test it should be created during the E2E test and deleted once the E2E is done.
It would be:
1 - Insert the dummy record (using a raw SQL query) before the test;
2 - Execute the E2E test using the dummy record inserted in step 1;
3 - Delete the dummy record created in step 1;
It would work for the remote E2E also, right?